developers/developers.md

```diff
@@ -12,11 +12,10 @@ ArduSub is hosted and maintained on [github](https://github.com/ardupilot/ardupi
 
 ## Compiling
 
-Before compiling ArduSub, first checkout the 3.6 (soon-to-be stable) version:
+Before compiling ArduSub, first checkout the 4.0 (stable) branch:
 
 ```
-git fetch --tags
-git checkout Sub-3.6
+git checkout Sub-4.0
```

Comment:
Use ArduSub-stable in place of Sub-4.0 ? This will be future proof and documentation will not need to change in the future